tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2023_01_11/unreal_assert_mac.pysnap

```diff
@@ -1,10 +1,10 @@
 ---
-created: '2025-02-25T14:42:17.298090+00:00'
+created: '2025-02-25T14:49:13.587085+00:00'
 creator: sentry
 source: tests/sentry/grouping/test_variants.py
 ---
 app:
-  hash: "57493ac3e558feffb778cf170a7fd986"
+  hash: "ecb890e5cd60dec2b626d500cc866de4"
```

Comment:
We now have two hashes:
<img width="379" alt="image" src="https://github.com/user-attachments/assets/f85a38b3-9274-4421-86d3-abb0987a5ef5" />